tests/unit_tests/all/plugins/test_parameter_disable.py

```diff
@@ -10,18 +8,6 @@
 from ert.shared.plugins import ErtPluginManager
 
 
-@pytest.mark.parametrize(
-    "input_string, expected", [("a", ["b", "c"]), ("a,b", ["c"]), ("a, b", ["c"])]
-)
-def test_parse_comma_list(tmpdir, monkeypatch, input_string, expected):
-    ert_mock = MagicMock()
-    ert_mock._observation_keys = ["OBSERVATION"]
-    ert_mock._parameter_keys = ["a", "b", "c"]
-
-    DisableParametersUpdate(ert_mock, storage=None).run(input_string)
-    assert ert_mock.update_configuration[0]["parameters"] == expected
```

Comment:
```suggestion
    assert [
        param.name for param in ert_mock.update_configuration[0].parameters
    ] == expected
```

Reply:
That makes sense, thanks.
I've pushed a fix.